dev-util/colm/colm-0.13.0.5.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="COmputer Language Manipulation"
+DESCRIPTION="Colm is a programming language designed for the analysis and \
+transformation of computer languages. Colm is influenced primarily by TXL. It \
+is in the family of program transformation languages."
+HOMEPAGE="https://www.colm.net/open-source/colm/"
```

Comment:
`HOMEPAGE` seems to lack an unified style. There are recipes with `/` at the end, and there are also recipe without it at the end (#2135)

Reply:
Here the / is perfect :)
The rule is that if you pass an URI to `curl --head` which is missing the trailing slash, then, if the server redirects you to the "corrected" URI, i.e. the same URI but with a slash appended at the end of the line, then this means we should have the trailing slash for the URI in `HOMEPAGE`.

Sometimes the URI does not need the slash, but the server won't complain if you add one.
That's the case for e.g. `https://github.com/haikuports/haikuports`. But since GitHub does not send a recirect for it, it is better to not add the slash in URIs for any project hosted on GitHub.